porcupine/plugins/text_google_search.py

```diff
@@ -0,0 +1,21 @@
+"""Search selected text on Google."""
+
+import tkinter as tk
+from porcupine import menubar, tabs
+import webbrowser
+
+def google_search(tab: tabs.FileTab) -> None:
+    try:
+        selected_text_tuple = tab.textwidget.tag_ranges(tk.SEL)
+        # Check if text is selected to search on google
+        if len(selected_text_tuple)>0:
```

Comment:
We probably shouldn't do the search if multiple lines of text are selected, or if the selected text consists entirely of whitespace characters (e.g. spaces or tabs, basically whatever `string.strip()` removes).